### DISCUSSION TITLE
access community.globaleaks.org on slack

### DISCUSSION THREAD

**Discussion by matcor73** [globaleaks/globaleaks-whistleblowing-software#3861] [Q&A] [answered]
HI, I would like to join the community on Slack, but if I'm not mistaken, I need a @globaleaks.org address. Who can I contact to obtain one? Thank you in advance
Matteo

**Comment by matcor73** [comment]
Anyone can help me please ?

**Comment by evilaliv3** [comment, marked as answer]
Helo @matcor73, we renewed our invite link; if you visit community.globaleaks.org you will be able to join with your own email!
